### PR TITLE
[17.0][MIG] base_multi_image: Migration to 17.0

### PR DESCRIPTION
**Pull request by psugne**

### CODE REVIEW

File: base_multi_image/README.rst [hunk 1]
```diff
@@ -138,28 +146,29 @@ Authors
...
+
+- Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+- Rafael Blasco <rafabn@antiun.com>
+- Jairo Llopis <yajo.sk8@gmail.com>
+- Sodexis <dev@sodexis.com>
+- Dave Lasley <dave@laslabs.com>
+- Shepilov Vladislav <shepilov.v@protonmail.com>
+- `Greenice <https://www.greenice.com>`__:
+- Fernando La Chica <fernandolachica@gmail.com>
+- Ugne Sinkeviciene <ugne@versada.eu>
```
Comment (oerp-odoo) [inline base_multi_image/README.rst]: I think LT letters should work here:)

File: base_multi_image/hooks.py [hunk 2]
```diff
@@ -29,9 +29,7 @@ def post_init_hook_for_submodules(cr, model, field):
     with cr.savepoint():
         column_exists = table_has_column(cr, table, field)
         if column_exists:
-            cr.execute(
-                "SELECT id FROM %(table)s WHERE %(field)s IS NOT NULL"
-            )  # pylint: disable=sql-injection
+            cr.execute("SELECT id FROM %(table)s WHERE %(field)s IS NOT NULL")  # pylint: disable=sql-injection
```
Comment (oerp-odoo) [inline base_multi_image/hooks.py]: better way would be to use explicit AsIs function, so you would not need to disable pylint.

**Review by oerp-odoo — APPROVED** [review]

File: base_multi_image/hooks.py (hunk 2 above)
Comment (psugne) [reply base_multi_image/hooks.py]: @oerp-odoo it's leftover. No need this `pylint` disable anymore, removed it.

File: base_multi_image/README.rst (hunk 1 above)
Comment (psugne) [reply base_multi_image/README.rst]: @oerp-odoo somehow I'm always skipping to use LT letters for easier readability when migrating Odoo/OCA modules :)

**Pull request merged**